media/admin/com_virtuemart/models/zasilkovna.php

```diff
@@ -203,7 +218,7 @@ public function publishShipmentMethods($ids, $value = 1)
      */
     public function getBranches() {
```

Comment:
V novém kódu prosím nepoužívat nikde branches - jsou tam i informace o HD přepravcích.

Reply:
Tato metoda nesouvisí s úkolem. Já jsem jen přejmenoval tabulku.

Reply:
Trvám na tom, nepřijde mi ok mít v getBranches select * from carriers :)

Reply:
HD přepravci jsou v Mordoru v tabulce branch. carrier_id je branch_id. Endpoint má URL končící na branch.xml?address-delivery. Metodu jsem vyhodil. Byla nadbytečná.

Reply:
Ještě bych zvážil úpravy saveBranchesXmlToDb a updateBranchesInfo.

Reply:
Jarda se mnou souhlasí, v novém a přepisovaném kódu nikde nepoužívat branch - je to vždy buď carrier nebo pickup point.